src/shapes.ts

```diff
@@ -91,6 +91,16 @@ export interface OpenFinWindow {
     id: number;
     name: string;
     uuid: string;
+    preloadState?: { // set if window has a preload script
+        src: String;
+        state:
+            'idle'| // initial state, not emitted, really here for internal use
```

Comment:
Super :neckbeard: but how do you feel about `'initial'` instead of `'idle'`? Just a thought